modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CppRestClientCodegen.java

```diff
@@ -187,14 +204,27 @@ public void processOpts() {
         additionalProperties.put("apiHeaderGuardPrefix", apiPackage.replaceAll("\\.", "_").toUpperCase());
         additionalProperties.put("declspec", declspec);
         additionalProperties.put("defaultInclude", defaultInclude);
+
+        if (additionalProperties.containsKey(SOURCE_OUTPUT_FOLDER_KEY))
+        {
+            sourceOutputFolder = additionalProperties.get(SOURCE_OUTPUT_FOLDER_KEY).toString();
+            supportingFiles.clear();
+            addSupportingFiles();
+
+            updateImportMapping();
```

Comment:
Shouldn't `updateImportMapping` be called before `addSupportingFiles` ?

Reply:
Why? Is there a dependency?

Reply:
Sorry, I think I was under the impression that `addSupportingFile` was already executing the templates, but it's not, it just registers the files in a list.
So no dependency indeed, ignore my previous comment.